.validate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "scripts": {
+    "lint": "jshint ."
+  },
+  "pre-commit": ["lint", "validate", "test"]
+}
```

Comment:
fair enough

On Thu, Jan 21, 2016 at 11:18 AM, Michael Garvin <notifications@github.com>
wrote:

> In .validate.json
> <https://github.com/AmpersandJS/ampersand-view/pull/154#discussion_r50439705>
> :
>
> > @@ -0,0 +1,6 @@
> > +{
> > +  "scripts": {
> > +    "lint": "jshint ."
> > +  },
> > +  "pre-commit": ["lint", "validate", "test"]
> > +}
>
> This was what an npm install added manually so I went with it. Updating
> precommit-hook is outside the scope of what I'm trying to do here, doing my
> best to avoid a rabbit hole of updating here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AmpersandJS/ampersand-view/pull/154/files#r50439705>.
>



-- 
Chris Dieringer
p: (503)358-8577
e: cdieringer@mrn.org
w: http://cdaringe.com